shell/edit/token.vue

```diff
@@ -273,6 +273,7 @@ export default {
       <div class="right">
         <button
           type="button"
+          data-testid="done_create_button"
```

Comment:
this one might need more context, to keep it unique. suggestion - `token_create_button`

Reply:
updated